test/features/labelDataDisplay.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Tester } from '@mighty-justice/tester';
+import { COMPONENT_GENERATORS, stringFactory } from '../factories';
+
+const fieldSets = [[stringFactory.build()]],
+  EDITABLE_COMPONENTS = ['EditableCard', 'EditableArrayCard'],
+  COMPONENTS = [
+    ...EDITABLE_COMPONENTS,
+    'ArrayCard',
+    'FormCard',
+    'SummaryCard',
+    'Card',
+    'Form',
+    'FormModal',
+    'FormDrawer',
+  ],
+  LAYOUTS = ['inline', 'vertical', 'horizontal'];
+
+function isForm(tester: any) {
+  return !!tester.find('button[type="submit"]').length;
+}
+
+describe('Renders', () => {
+  COMPONENTS.forEach(componentName => {
+    LAYOUTS.forEach(layout => {
+      it(`Displays ${layout} for ${componentName}`, async () => {
+        const { ComponentClass, propsFactory } = COMPONENT_GENERATORS[componentName],
+          props = propsFactory.build({
+            fieldSets,
+            layout,
+          }),
+          tester = await new Tester(ComponentClass, { props }).mount();
+
+        expect(tester.find(`.fields-ant-info-row-${layout}`).length);
+        expect(tester.find(`.fields-ant-info-label-layout-${layout}`).length);
+      });
+    });
+  });
+
+  COMPONENTS.forEach(componentName => {
+    LAYOUTS.forEach(layout => {
+      it(`Displays colon properly for ${layout} ${componentName}`, async () => {
+        const { ComponentClass, propsFactory } = COMPONENT_GENERATORS[componentName],
+          props = propsFactory.build({
+            fieldSets,
+            layout,
+            colon: true,
+          }),
+          tester = await new Tester(ComponentClass, { props }).mount();
+
+        if (layout === 'vertical') {
+          expect(tester.find('.fields-ant-info-label-no-colon').length);
+        } else {
+          expect(tester.find('.fields-ant-info-label-colon').length);
+        }
+      });
+    });
+  });
+
+  EDITABLE_COMPONENTS.forEach(componentName => {
+    LAYOUTS.forEach(layout => {
+      it(`Displays ${layout} when reading and writing for ${componentName}`, async () => {
+        const { ComponentClass, propsFactory } = COMPONENT_GENERATORS[componentName],
+          props = propsFactory.build({
+            fieldSets,
+            layout,
+          }),
+          tester = await new Tester(ComponentClass, { props }).mount();
+
+        expect(isForm(tester)).toBe(false);
+        expect(tester.find(`.fields-ant-info-row-${layout}`).length);
```

Comment:
Does this actually test anything without a `.toBe(1)` in it? Like does it default to checking if the value is truthy? Or does it need something added to the end?

Reply:
Ah yeah that's a good catch. I assumed that expect would default to false for 0 which is was not the case. I updated the tests and removed the colon test for the `CARD_COMPONENTS` because fields ant handles the colon with an `::after`. Do you think it makes sense still to add a class like `fields-ant-form-label-${layout}` to the form and test it?

Reply:
Let's leave it for now and add it if we need it, unless you think it's good for uniformity and consistency.

Reply:
Actually, I added a PR for this and to test component defaults so if it's overkill I can easily revert a commit if needed